lfc-api/LookingForConcerts/lfc_backend/views.py

```diff
@@ -734,20 +738,22 @@ def get_tags(request, search_str):
         'format' : 'json',
         'language' : ['en', 'tr'],
         'search' : query,
-        'type':'item'
+        'type'   :'item'
     }
 
     r = requests.get(API_ENDPOINT, params = params)
     json_response = r.json()['search']
     lenght =  len(json_response)
-
+    #print(json_response)
+    
     tags = []
     for i in range(lenght):
         if 'description' in json_response[i]:
             if any(re.findall(r'music|genre', json_response[i]['description'], re.IGNORECASE)):
-                value = json_response[i]['label']
+                value   = json_response[i]['label']
                 context = json_response[i]['description']
-                t = '{"value":"'+value.replace('"','')+ '","context":"'+context.replace('"','')+'"}'
+                url     = json_response[i]['concepturi']

```

Comment:
extra space?

Reply:
This my convention of writing code. I try to align variables as much as possible.